### ISSUE TITLE
Continue world generation beyond the world gen % restriction after queue is filled

### ISSUE THREAD

**Issue by sathya-pramodh** [KingContaria/seedqueue#8]
Eg: If max gen % is set to 5% and once queue fills up, it continues generating the remaining seeds in queue?

Considerations for this:
1. Might degrade performance if queue fills up while in game... (Could be made an opt-in config for that).
2. Would generate seeds the same way as it generated to fill the queue initially (i.e if someone wants to have 3 seeds generating while in-game, we would respect that even for the further loading post queue fill.. this could also fix consideration #1).
3. Maybe even think about setting lower thread priorities for these types of seeds (could be a config option as well).

That's all I could think of, would love to hear thoughts on this :)

**Comment by KingContaria** [comment]
Makes sense as an option, I'm unsure if it should be under Queue Settings or somewhere else.
It makes sense grouped with the Queue Settings but I would also like those to be very simple and as few as possible.

I've marked this as a good-first-issue for new contributors so I don't have to figure out what I would name an option like this :P

**Comment by sathya-pramodh** [comment]
Mhm i agree.... Queue Settings makes sense. I'll figure out a good way to do this :D

**Comment by sathya-pramodh** [comment]
Coming to justin's issue mention, I could just cross that bridge when I get to it ig. Lemme see tho... I will try to make it as future-proof as possible..

**Comment by KingContaria** [comment]
Closed by https://github.com/KingContaria/seedqueue/pull/56
